api/v1/model_handlers.go

```diff
@@ -474,6 +490,76 @@ func (a *Api) podMetrics(request *restful.Request, response *restful.Response) {
 	response.WriteEntity(exportTimeseries(timeseries, new_stamp))
 }
 
+type AggregatedMetricPoints []AggregatedMetricPoint
+
+func (s AggregatedMetricPoints) Len() int {
+	return len(s)
+}
+func (s AggregatedMetricPoints) Swap(i, j int) {
+	s[i], s[j] = s[j], s[i]
+}
+func (s AggregatedMetricPoints) Less(i, j int) bool {
+	// we want the descending order
+	return s[j].Timestamp.Before(s[i].Timestamp)
+}
+
+// podMetrics returns a metric timeseries for a metric of the Pod entity.
+func (a *Api) podMetricsSum(request *restful.Request, response *restful.Response) {
```

Comment:
In the end I decided to try with `[][]store.TimePoint`. It seems clearer to get all metrics and sum them elsewhere than summing them up inside the db, under lock. Please take a look.